frontend/src/components/config-details.ts

```diff
@@ -367,7 +388,14 @@ export class ConfigDetails extends LiteElement {
               <ul>
                 ${additionalUrlList.map(
                   (seed) =>
-                    html`<li>${typeof seed === "string" ? seed : seed.url}</li>`
+                    html`<li>
+                      <a
+                        class="text-primary hover:text-indigo-400"
+                        href="${typeof seed === "string" ? seed : seed.url}"
+                        target="_blank"
+                        >${typeof seed === "string" ? seed : seed.url}</a
+                      >
+                    </li>`
```

Comment:
You could store the seed url as a variable here:
```
(seed) => {
  const seedUrl = typeof seed === "string" ? seed : seed.url;
  return html`<li>
      <a
        class="text-primary hover:text-indigo-400"
        href="${seedUrl}"
        target="_blank"
        >${seedUrl}</a
      >
    </li>`;
}
```

Reply:
Hank being bad at code strikes again ;P

Do you make variables every time you have to use something more than once?  I feel like I try to do it when there's more than 3 usages of the data, but obviously it's cleaner and (probably) better to do it when there's only 2 with the bonus of being able to use it later?